packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -44,6 +43,29 @@ export const expandVariants = styleVariants({
   false: [sprinkles({ width: 'max-content', maxWidth: 'maxContent' })],
 });
 
+export const displayVariants = styleVariants({
+  outlined: [sprinkles({ borderWidth: '$sm' })],
+  standard: { borderWidth: 0 },
+});
+
+export const inlineVariants = styleVariants({
+  true: [
+    sprinkles({
+      display: 'flex',
+      flexDirection: 'column',
+    }),
+    {
+      '@media': {
+        [`screen and ${breakpoints.md}`]: {
+          alignItems: 'flex-start',
+          flexDirection: 'row',
+        },
```

Comment:
I believe you can put this in sprinkles

Reply:
you can use this syntax ```
```suggestion
sprinkles({
alignItems: {
md: "flex-start",
}), 
flexDirection: {
md: "row"
}
})
```

Reply:
Thank you! I'll try that syntax :)